python/ray/_private/usage/usage_lib.py

```diff
@@ -620,6 +620,7 @@ def get_extra_usage_tags_to_report(gcs_client) -> Dict[str, str]:
             extra_usage_tags[key] = value.decode("utf-8")
     except Exception as e:
         logger.info(f"Failed to get extra usage tags from kv store {e}")
+    print(extra_usage_tags)
```

Comment:
Should this be removed before merge?

Reply:
https://github.com/ray-project/ray/pull/32322 :doge:

Reply:
can we make sure to cherry pick this in to 2.3 as well?